src/libxtp/davidsonsolver.cc

```diff
@@ -397,6 +385,10 @@ Eigen::MatrixXd DavidsonSolver::gramschmidt(const Eigen::MatrixXd &A,
   Eigen::MatrixXd Q = A;
   for (Index j = nstart; j < A.cols(); ++j) {
     Q.col(j) -= Q.leftCols(j) * (Q.leftCols(j).transpose() * A.col(j));
+    Q.col(j).normalize();
+    // two is enough GS
+    // http://stoppels.blog/posts/orthogonalization-performance
+    Q.col(j) -= Q.leftCols(j) * (Q.leftCols(j).transpose() * Q.col(j));
```

Comment:
did you notice any difference with that ? Cause the numbers were exactly the same when I tried that ... So if it does nothing I would remove the second part. Or at least make it another options (yeah ! another option)

Reply:
Actually, yes I have seen a difference in the overlap matrix between the vectors.